### ISSUE TITLE
Add way to use classic and minmax in parallel

### ISSUE THREAD

**Issue by gdt** [merbanan/rtl_433#2645]
The basic idea is that if either decodes, you have the data, and there's no reason to have a global choice.

**Comment by ondrej-zary** [comment]
ELV KS200/KS300 (also sold as a part of Welltech 40616/TW69) has the same problem - the new default minmax decoder misidentifies the packages as FSK. With -Y classic, the packages are identified as OOK and decoded properly.

**Comment by zuckschwerdt** [comment]
Yes, there are ideas to run multiple optimized demods to get better results. This will be a major architecture change and an estimate for this to land is ~half a year.
